.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Please add node_modules folder in this file. Because when you start adding javascript packages you may accidentally push its content to GitHub. And node_modules folder should not be pushed to GitHub.